css/search.css

```diff
@@ -0,0 +1,61 @@
+* {
+  font-family: "Source Sans Pro", sans-serif;
+}
+.background-img {
+  position: static;
+  display: flex;
+  justify-content: center;
+  margin-top: 150px;
+  opacity: 0.6;
+}
+
+header {
+  background-color: white;
+  position: fixed;
+  left: 0;
+  top: 0;
+  padding-top: 10%;
+  width: 100%;
+  height: 40px;
+  font-size: 19px;
+  font-weight: bold;
+  text-align: center;
+  box-shadow: 0px 0px 14px 2px #dcd6d6;
+  z-index: 98;
+}
+/* footer style */
+.footer-icon img {
+  display: flex;
+  width: 20px;
+  height: 20px;
+  margin-left: 12px;
+}
+.footer-icon:active {
+  color: #ff647c;
+}
+.footer-text {
+  font-size: 16px;
+  margin-top: -2px;
+  margin-left: -2px;
+}
+.footer-icon {
+  text-decoration: none;
+  color: #afaaaa;
+  display: flex;
+  flex-direction: column;
+}
+footer {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: white;
+  position: fixed;
+  color: #dcd6d6;
+  width: 100%;
```

Comment:
https://github.com/GSG-FC02/CovidApp/blob/f48ee67d90dfbbdbfe2fa798a440a4d0923307b3/css/search.css#L53-L58

repeated width, delete one of them :